### ISSUE TITLE
Add documentation on adding the runtime variable for specific languages (go, Typescript, Rust)

### ISSUE THREAD

**Issue by eleijonmarck** [helix-editor/helix#85]
Hello!
I love the project! :heart: 

However I cannot seem to get my head around setting the runtime variable to get syntax highllighting and stuffz for my Go projects or Typescript/Javascript or Rust projects.

I move the `hx` binary into the different projects, however it does not seem to pickup the runtime.
Could you provide a example for let's say a Go project using go.mod and explicitly setting the HELIX_RUNTIME variable for that?

**Comment by pickfire** [comment]
Runtime variable?

**Comment by kevinsjoberg** [comment]
An example from my personal configuration.

```console
/tmp $ which hx
/Users/kevinsjoberg/bin/hx
```

```console
/tmp $ ls -l $HOME/.config/helix/
total 8
-rw-r--r--  1 kevinsjoberg  staff  1560 Jun  2 22:09 helix.log
drwxr-xr-x@ 3 kevinsjoberg  staff    96 Jun  3 04:07 runtime
```

```console
/tmp $ echo $HELIX_RUNTIME
/Users/kevinsjoberg/.config/helix/runtime
```

**Comment by eleijonmarck** [comment]
@kevinsjoberg thx for the clear example.
I downloaded the release, but did not know where to put the things. :dancers:

**Comment by eleijonmarck** [comment]
![image](https://user-images.githubusercontent.com/7727602/120636035-70ed2080-c46d-11eb-98ac-5121a317fc9a.png)
I now get highlighting and goto definition and stuffz !!

**Comment by pickfire** [comment]
https://docs.helix-editor.com/install.html#build-from-source

Ah, so you mean

> Now copy the runtime/ directory somewhere. Helix will by default look for the runtime inside the same folder as the executable, but that can be overriden via the HELIX_RUNTIME environment variable.

I was confused about that either. No wonder I didn't get syntax highlighting. I still don't get it.

**Comment by eleijonmarck** [comment]
Yea I was thinking to update the docs, with this explicit information.
So that It gets easy to open `hx` and start to play around and making it more frictionless.

**Comment by archseer** [comment]
The binary looks for `runtime/` where the executable is located, or for a `HELIX_RUNTIME` override. https://github.com/helix-editor/helix/blob/abe8a83d8e0adeb83c188b28501176e17f4c4fbf/helix-core/src/lib.rs#L47-L57

If running this in development via `cargo run` I recommend using an `.envrc` file to automatically set the var. We could also try detecting that (if debug mode) and setting it to the correct path.

For packaged installs, I recommend unpacking the files somewhere, then making a `hx` wrapper that sets the var and runs the binary. [Example from Arch Linux](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=helix-bin)

**Comment by eleijonmarck** [comment]
I agree with the package install that sets the VAR to some default where the default runtime is set on all comp.

@archseer 
For releases or anything in the documentation, we could add commands that would make it easier to add the ENV to point to a `~/.config/helix` by default if not specified.

Maybe this is what you mean by the package installs. But would be great to get this in right away.

Package installer for me did not work:
I did not get it to install for my Ubuntu 20.04 distro though. Maybe this is just me

**Comment by kirawi** [comment]
I don't think it correctly detects it on Windows.

**Comment by archseer** [comment]
I think this should be solved in 0.2.0, we have more reasonable defaults now.
